src/otk_external_osbuild/command/make_depsolve_dnf4_rpm_stage.py

```diff
@@ -5,7 +5,14 @@
 
 def root(input_stream: TextIO) -> None:
     data = json.load(input_stream)
-    tree = data["tree"]["packageset"]["const"]["internal"]
+    tree = data["tree"]
+    pkgs = tree["packageset"]["const"]["internal"]
+
+    # allow passing on stage options
+    opts = tree.get("options", {})
+
+    # perhaps gpgkeys should move *under* options?
```

Comment:
Yes, I think we should be consistent here, either we expose all rpm stage options under `options` or we hand pick like we did for rpm-keys. Given that our philosophy so far is that we give people the full osbuild stages with all options it seems fine to expose "options" and put gpgkeys under it. It will require a bit of tweaking in the existing manifests but that seems worth the extra consistentcy.